nautobot/extras/tests/test_customfields.py

```diff
@@ -2083,6 +2083,30 @@ def test_delete_custom_field_data_task(self):
         self.assertEqual(oc_list[0].change_context_detail, "delete custom field data")
         self.assertEqual(oc_list[0].user, self.user)
 
+    def test_clear_custom_field_data_task(self):
```

Comment:
Should we add tests for situations where `content_types.add()/.delete()/.clear()` did not actually change the custom fields and assert that no object changes are generated?

Reply:
Really we should have tests to assert that the tasks aren't triggered at all in those cases, but I already spent more time on this issue today than I'd meant to spend given it was an emergent bug report rather than planned work.